sdk/lib/atl/atlconv.h

```diff
@@ -322,7 +322,7 @@ class CW2AEX
             }
 
             // A complex UTF-8 string might come here
-            cchMax = 3 * cch + 1;
+            cchMax = 4 * cch + 1;
```

Comment:
Doesn't WideCharToMultiByte return the expected number of WCHAR units for the string?
> If successful, returns the number of bytes written to the buffer pointed to by lpMultiByteStr. _**If the function succeeds and cbMultiByte is 0, the return value is the required size, in bytes, for the buffer indicated by lpMultiByteStr.**_

Reply:
Fixed by using `WideCharToMultiByte`.